### DISCUSSION TITLE
LDAPサーバへの接続設定ができません。

### DISCUSSION THREAD

**Discussion by ai-sato01** [SeciossOpenSource/LISM#3] [Q&A]
「LDAPサーバへの接続設定」でPasswordを入力する箇所があり（★の箇所）
BindDNのPasswordを入力する必要があるかと思いますが、事前に設定したのは
OpenLDAPの管理者パスワードのみBindDNのパスワード設定はしていません。
設定するにはどの様にすればよいでしょうか。

＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝
LDAPサーバの設定を行います。
LDAP-pathを入力してください。(default: ldap://localhost)
ldaps://sime.ldap.com
BaseDNを入力してください。(default: dc=example,dc=com)
dc=sime,dc=ldap,dc=com
BindDNを入力してください。(default: cn=Manager,dc=example,dc=com)
cn=Manager,dc=sime,dc=ldap,dc=com
Passwordを入力してください。★
LDAPサーバへの接続設定が完了しました。
＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝＝

**Comment by kaoru-sekiguchi** [comment]
最初にsecioss_ldif内のdb.ldifのデータを登録したかと思いますが、その中のolcRootPWに設定したパスワードを設定して下さい。
oclRootPWを変更していない場合は、設定したいパスワードの値に書き換えて、再度db.ldifのデータで更新を行って下さい。
`# ldapmodify -Y EXTERNAL -H ldapi:// -f db.ldif`
